config/config.go

```diff
@@ -536,6 +536,8 @@ type MempoolConfig struct {
 	WalPath   string `mapstructure:"wal_dir"`
 	Size      int    `mapstructure:"size"`
 	CacheSize int    `mapstructure:"cache_size"`
+	// Maximum size of all txs in the mempool in bytes
+	MaxBytes int `mapstructure:"max_bytes"`
```

Comment:
What is a rule of thumb regarding integer types? When we use int and when we use fixed types such as int16, int32 etc?

Reply:
int32 can represent up to 2GB, so it's not enough. This can be uint, but then we'll need to type cast again in mempool